server/form-pages/apply/access-and-healthcare/accessNeeds.ts

```diff
@@ -63,7 +63,10 @@ export default class AccessNeeds implements TasklistPage {
   }
 
   next() {
-    return this.body.additionalNeeds.includes('mobility') ? 'access-needs-mobility' : 'covid'
+    if (this.body.additionalNeeds.includes('mobility')) return 'access-needs-mobility'
+    if (this.body.additionalNeeds[0] === 'none') return 'covid'
```

Comment:
Nice catch - we could probably improve on this with some frontend JS to disable the other checkboxes if the user selects `none`, but we should have this server-side catch anyway